### PR TITLE
Potential Nav

### PR DESCRIPTION
**Pull request by idalithb**
-potential left-side navigation structure

### CODE REVIEW

File: website/pages/en/network/billing.mdx
Comment (saihaj) [inline website/pages/en/network/billing.mdx:1]: we should setup a redirect for this page so old paths don't see 404

File: website/pages/en/_meta.js [hunk 1]
```diff
@@ -3,19 +3,24 @@ export default {
...
+  firehose: '',
+  substreams: '',
   '---2': {
     type: 'separator',
   },
-  '###1': {
+  '###2': {
     type: 'heading',
-    title: 'Subgraphs',
+    title: 'Developers',
```
Comment (benface) [inline website/pages/en/_meta.js:23]: Hmm, interesting change. Seems a bit strange to have a "Developing" section under a "Developers" heading, haha. Are those sections currently not all about subgraphs?

Comment (benface) [reply website/pages/en/_meta.js:23]: Also a couple of those section titles should probably be changed to include the word "subgraphs" now that it's not in the heading, like "Managing" and maybe "Deploying" and "Publishing" too.

File: website/pages/en/network/billing.mdx
Comment (benface) [reply website/pages/en/network/billing.mdx:1]: Yes, and I just realized that I approved #741 and it got merged without redirects. 🥲 @idalithb, are you able to add those, maybe all in this PR?

Comment (benface) [reply website/pages/en/network/billing.mdx:1]: @idalithb – I've done it in #753, if you want to take a look at the diff and the comments there.

File: website/pages/en/_meta.js [hunk 2]
```diff
@@ -3,19 +3,24 @@ export default {
...
+    title: 'Overview',
+  },
   about: '',
   network: 'The Graph Network',
-  sunrise: 'Post-Sunrise Upgrade FAQ',
-  billing: '',
-  glossary: '',
   tokenomics: 'Tokenomics',
-  arbitrum: 'Scaling with Arbitrum',
+  contracts: '',
```
Comment (benface) [inline website/pages/en/_meta.js:13]: I could be wrong, but I feel like Tokenomics should be under "The Graph Network". And maybe even Contracts?

File: website/pages/en/_meta.js [hunk 3]
```diff
@@ -3,19 +3,24 @@ export default {
...
+  firehose: '',
+  substreams: '',
   '---2': {
     type: 'separator',
   },
-  '###1': {
+  '###2': {
     type: 'heading',
-    title: 'Subgraphs',
+    title: 'Subgraph Developers',
```
Comment (benface) [inline website/pages/en/_meta.js:23]: Nice solution 👍

File: website/pages/en/network/_meta.js [hunk 4]
```diff
@@ -1,9 +1,9 @@
 export default {
   overview: 'Overview',
   benefits: 'Benefits',
+  billing: '',
```
Comment (benface) [inline website/pages/en/network/_meta.js:4]: I'm confused as to why Billing was moved to The Graph Network.

File: website/pages/en/network/billing.mdx
Comment (benface) [reply website/pages/en/network/billing.mdx:1]: I was about to resolve this, but a redirect still needs to be added for the Billing page, if we indeed want to move it (I left a comment about that [here](https://github.com/graphprotocol/docs/pull/744#discussion_r1742205053)).

**Pull request closed without merging**